src/ludic.rs

```diff
@@ -0,0 +1,98 @@
+// https://rosettacode.org/wiki/Ludic_numbers
+
+const ARRAY_MAX: usize = 25000;
+const LUDIC_MAX: usize = 2100;
+
+fn main() {
+
+    // Variable that will hold all the ludic numbers
+    let mut result: Vec<i32> = vec![];
+
+    // Array that will hold the first array with 25.000 numbers starting with number 2
+    let mut array: [i32; ARRAY_MAX] = [0; ARRAY_MAX];
+
+    // Construct the first array
+    let mut first_number: i32 = 2;
+    for i in 0..array.len() {
```

Comment:
It looks like here you're creating a vector that is `[2,3,4,5,6...,25000]`.

A preferable way might be to use the [`Range`](https://doc.rust-lang.org/std/ops/struct.Range.html) type like so: `(2..(ARRAY_MAX+2)).collect::<Vec<_>>()`.